README.md

```diff
@@ -37,15 +37,16 @@ println(actualResults["2.0.10"])
 
 ### Changing the Maven repository
 
-By default the `ReleaseHistoryService` uses Maven Central to retrieve release history information. To use another repository set the `repositoryUrl` property to the appropriate value.
+By default the `ReleaseHistoryService` uses Maven Central to retrieve release history information. To use another repository pass the value into the constructor.
 
 For example, here's how to create an instance of the service that communicates with the repository used by the Spring project.
 
 ```kotlin
-val service = ReleaseHistoryService()
-service.repositoryUrl = "https://repo.spring.io/artifactory/release/"
+val service = ReleaseHistoryService("https://repo.spring.io/artifactory/release/")
 ```
 
+> :warning: When you specify an alternative repository, Maven Central will still be contacted to retrieve the information that it has for the package in question. This works around issues with some Maven repository proxies not preserving release history information. For each package version that is queried, the oldest date when be used.
```

Comment:
Should this say "oldest date will be used"?

Reply:
Good catch. I've fixed that.